### DISCUSSION TITLE
C++ DOM framework library for MNX?

### DISCUSSION THREAD

**Discussion by rpatters1** [w3c/mnx#360] [General]
Is anyone aware of an open source DOM library (either in C++ or usable by C++) that read/writes MNX files?

**Comment by paul-bayleaf** [comment]
Not specifically for MNX.  But any good JSON C++ library should work, there are several.  I've been using https://github.com/nlohmann/json for reading and writing MNX, it works very well.

**Comment by rpatters1** [comment]
OK. I am just getting started with MNX, so it is difficult for me to evaluate the value of a dedicated MNX framework vs. a generic solution. Thank you for the suggestion, though. It looks like a promising starting point.
